app/Http/Controllers/EmployeeController.php

```diff
@@ -46,6 +49,11 @@ function ($q) use ($search) {
         return Response::json($response);
     }
 
+    public function create(){
+        $companies = Company::all();
+        return Response::json($companies);
+    }
+
     public function store(Request $request){
 
         $validated = $request->validate([
```

Comment:
make two validation separately one is for `Employee` another one is for `BankAccount`

Reply:
i tried this as a first approach...but i found an issue in this approach that is whenthe validation runs if one model fails it doesnt check the other model and only sends the errors of one model....to flash session when giving alert messages...so when giving alert feedbacks it doesnt show all the errors at once which is a bad UX, i have attached the behaviour as click...please find the attachment 

https://github.com/user-attachments/assets/aa3f641c-647e-4d6a-b083-6978624a9ba4
 @Raza9798

Reply:
![image](https://github.com/user-attachments/assets/bd923f86-9e15-41f8-9b6c-14c1d8c225e0)

this is the way i have implmented...if there is a different apporach tht can display all model error together can you give some hints please @Raza9798

Reply:
> i tried this as a first approach...but i found an issue in this approach that is whenthe validation runs if one model fails it doesnt check the other model and only sends the errors of one model....to flash session when giving alert messages...so when giving alert feedbacks it doesnt show all the errors at once which is a bad UX, i have attached the behaviour as click...please find the attachment
> 
> https://github.com/user-attachments/assets/aa3f641c-647e-4d6a-b083-6978624a9ba4 @Raza9798

You can use error handler for resolve this issue @axnish27 `Eg: try catch`